target/linux/mediatek/dts/mt7986a-zyxel-ex5601-t0-ubootmod.dts

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+/*
+ * Copyright (C) 2021 MediaTek Inc.
+ * Author: Sam.Shih <sam.shih@mediatek.com>
+ */
+
+/dts-v1/;
+#include "mt7986a.dtsi"
+#include "mt7986a-zyxel-ex5601-t0-common.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Zyxel EX5601-T0 ubootmod";
+	compatible = "zyxel,ex5601-t0-ubootmod", "mediatek,mt7986a";
+};
```

Comment:
Since https://github.com/openwrt/openwrt/commit/572ea680705311a5ba08cb0f5da4c7dc2b792fd3#diff-0c89bd92e2a03d26907d885e84652e23bed36ef4ffcdbb1363d696c3e85b33b6 it is mandatory to include a `memory` node in device tree.
```suggestion
	memory@40000000 {
		device_type = "memory";
		reg = <0x40000000 0x20000000>;
	};
};
```